pytg/fix_msg_array.py

```diff
@@ -41,12 +42,19 @@ def fix_message(message):
 
 
 def fix_peer(peer):
+	# rename peer_type => type
+	if peer["peer_type"]:
+		peer["type"] = peer["peer_type"]
+		del peer["peer_type"]
+
 	# add cmd field
 	if peer["type"] == TGL_PEER_ENCR_CHAT:
 		assert peer["print_name"].startswith(ENCR_CHAT_PREFIX)
 		peer["cmd"] = peer["print_name"]
+	elif peer["type"] == TGL_PEER_CHANNEL:
+		peer["cmd"] = u("%s#id%d") % (peer["type"], peer["peer_id"])
 	else:
-		peer["cmd"] = peer["type"] + u("#") + u(str(peer["id"]))
+		peer["cmd"] = u("%s#%d") % (peer["type"], peer["peer_id"])
```

Comment:
Well, there is a problem here. In test branch, what used to be "id" is renamed with "peer_id" and "id" now is something else. But I'm not sure whether it's OK for master branch.